### DISCUSSION TITLE
Extensive contributor credits section tied with the game entry

### DISCUSSION THREAD

**Discussion by julkasalsa** [RetroAchievements/RAWeb#689] [Ideas]
This would list every person who worked on the set, and that include:
- code developers and debuggers (who made, improved or fixed the code);
- achievement designers (who brought the idea for more creative achievements);
- graphic designers (who made any images related to the set and game page);
- beta testers (who played the game after the set was published, revised or fixed and shared their comments this credit could be granted by personal recommendation of developer);
- special thanks (antything else that is creditworthy);
It could get it's own subpage for each game entry, or hidden in collapsable envelope.

**Comment by Jamiras** [comment]
#1096 has some mockups for artist credit
